gradio/routes.py

```diff
@@ -1567,7 +1567,7 @@ def read_main():
         else (
             ssr_mode
             if ssr_mode is not None
-            else bool(os.getenv("GRADIO_SSR_MODE", "False"))
+            else os.getenv("GRADIO_SSR_MODE", "False").lower() == "true"
```

Comment:
This behavior is consistent with: https://github.com/gradio-app/gradio/blob/74f3b9ded1ce0ff6f2f0dbcb113edd64b7bb8f60/gradio/blocks.py#L2413